README.md

```diff
@@ -56,6 +56,56 @@ module.exports = async x => x ** 2;
 module.exports = x => Promise.resolve(x ** 2);
 ```
 
+### Streams (experimental)
+
+Streaming functions can be created by setting the `$interactionModel` property on the function to `node-streams`.
+The function will then be invoked with two arguments, an `input` [Readable Stream](https://nodejs.org/dist/latest-v8.x/docs/api/stream.html#stream_class_stream_readable) and an `output` [Writeable Stream](https://nodejs.org/dist/latest-v8.x/docs/api/stream.html#stream_class_stream_writable).
+Both streams are object streams. Any value returned by the function is ignored, new messages must be written to the output stream.
+
+```js
+// echo.js
+module.exports = (input, output) => {
+    input.pipe(output);
+};
+module.exports.$interactionModel = 'node-streams';
+```
+
+Any npm package that works with Node Streams can be used.
```

Comment:
That it works with mississippi isn't interesting, it's just an example.

Reply:
i'm just saying that it's hard to infer what mississippi is without being told or without looking it up, and because its kindof a weird name it's distracting.